src/gamemode_defense.cpp

```diff
@@ -500,16 +498,15 @@ void defense_game::setup()
     } );
     ui.mark_resize();
 
-    int selection = 1;
-    int selection_max = 20;
+    int selection = 0;
```

Comment:
changing to zero based indexing caused a lot of churn below, but also simplified the arithmetic and is more consistent with other UI implementations